action.yml

```diff
@@ -48,6 +48,10 @@ inputs:
     description: Directory where go.mod is located
     required: true
     default: .
+  go_version:
+    description: Specify your Go version
+    required: true
+    default: 1.14.2
```

Comment:
`1.14.2` is the latest version at this time, but not latest version in future. I don't want to release action every time there is a new go version.

This is hard coding in another sense.

I prefer getting the latest version when performing an action. I think it's natural to use the latest version if `go_version` was omitted

This is a snippet to get the latest version of Go.

```bash
$ curl -s https://api.github.com/repos/golang/go/git/refs/tags | jq --raw-output '.[].ref | select(test("^refs/tags/go[0-9.]+$"))' | tail -n 1 | sed 's!refs/tags/go!!'
1.14.2
```

Reply:
Thanks for the command, it's much better